pkg/channel/distributed/controller/kafkasecret/controller.go

```diff
@@ -121,7 +122,7 @@ func enqueueSecretOfKafkaChannel(controller *controller.Impl) func(obj interface
 				secretName := labels[constants.KafkaSecretLabel]
 				if len(secretName) > 0 {
 					controller.EnqueueKey(types.NamespacedName{
-						Namespace: commonconstants.KnativeEventingNamespace,
+						Namespace: system.Namespace(),
 						Name:      secretName,
```

Comment:
Since we have the environment.SystemNamespace in the `NewController()` function above which calls this, can we pass it in as a parameter instead of re-loading from the GO environment again?

Reply:
Excellent idea